### PR TITLE
Foreach (#1), automatic wrapping, feedback message, feature flags

### PR DESCRIPTION
**Pull request by emragins**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
   "name": "strengthify",
-  "version": "0.4.2",
-  "homepage": "https://github.com/MorrisJobke/strengthify",
+  "version": "0.5.0",
```
Comment (MorrisJobke) [inline bower.json]: The changelog lists 0.5.2

File: bower.json [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
+  "homepage": "https://github.com/eve-corp/strengthify",
```
Comment (MorrisJobke) [inline bower.json]: I think it would be best to keep the URL as before ;)

**Comment by MorrisJobke** [comment]
@emragins Thanks for your pull request - beside my small comments this looks already good. I will try to test this soon and then merge it.

**Comment by MorrisJobke** [comment]
* [ ] drawBars seems to be not disablable
* [ ] drawMessage seems to be true by default
* [ ] somehow the meter is draw multiple times with multiple calls to the strengthify method

I used https://gist.github.com/MorrisJobke/fff6517861a36dad7860578646da48c9 to test this. I will add this to the repo once this PR is ready.

**Comment by MorrisJobke** [comment]
@emragins Could I ask you to check the above points? That would be really awesome :)

**Comment by emragins** [comment]
Yep.  Can do.  And no worries about the homepage.  When I originally made the changes I was imagining it might end up too divergent to merge back in :)  Thanks for taking a look at it!

**Comment by emragins** [comment]
I've made changes and added a couple additional tests / minor tweaks as well.  Here's the file I tested with: https://gist.github.com/emragins/cb7f345b9a97be46b5f83298852b89fe 

Not sure how to go about checking for disposability.... It's not something I've ever worried about in javascript, though I probably should.

**Comment by MorrisJobke** [comment]
I like it 👍 

I tested it and it works 👯 

bildschirmfoto 2016-04-06 um 09 26 51

**Pull request merged**